plugins/bufbuild/es/v0.2.1/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# syntax=docker/dockerfile:1.4
+FROM node:18.12.0-bullseye-slim AS build
```

Comment:
Bumped this to latest LTS release (thanks for suggestion Philip).

https://nodejs.org/en/blog/release/v18.12.0/

Reply:
We'll also want to bump the version below used at runtime.